Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/resources/static/processdiscoverer/zul/processDiscoverer.zul

```diff
@@ -134,6 +134,11 @@
                         'ap-mc-icon-play',
                         'ap-mc-icon-pause');
 
+    Ap.pd.primaryOverlay = 'FREQUENCY';
+    Ap.pd.setPrimaryOverlay = function(measureType) {
+      Ap.pd.primaryOverlay = measureType;
+    }
```

Comment:
Can we move this function definition to processdiscoverer/graph.js instead of making it lonely here while the rest of all PD functions are there? Putting some functions in the zul file makes it hard to manage PD code, where it uses Ap.pd.primaryOverlay === 'COST', but magically setting value for Ap.pd.primaryOverlay is located in a ZUL file.

Reply:
Can we make primaryOverlay part of the _private variable set of PD? This is to indicate that it is private.

![image](https://user-images.githubusercontent.com/12888841/186075067-e1c55621-5c85-4325-9d42-4a82c08ada80.png)

Reply:
In the _private variable set above, for primaryOverlay, we can set it to 'FREQUENCY' in PD constructor (the same as other instance variables). It is more consistent than setting it alone after the constructor in the ZUL file.